src/endpoints/createCategoriesEndpoint.js

```diff
@@ -25,7 +25,7 @@ export default (baseUrl: string): Endpoint<ParamsType, CategoriesMapModel> =>
       const categories = json
         .map(category => {
           return new CategoryModel({
-            id: category.id,
+            root: category.id === 0,
```

Comment:
This is always `false` as far as I can see, so we can hard code it.

Reply:
oh you are right! lolz